README.md

```diff
@@ -21,6 +21,25 @@ Note: there is also an online version, which can be found at [my-mind.github.io]
   * ~~Open index.html in your webbrowser~~ currently not working
 * Done! If need be, you can find the manual [here](https://github.com/ondras/my-mind/wiki)
 
+## Docker
+To dockerize My Mind 
+
+1. Clone the repo 
+   
+   `git clone git@github.com:ondras/my-mind.git`
+
```

Comment:
Possibly better to use the HTTPS URI:

git clone https://github.com/ondras/my-mind.git